### PR TITLE
Add managed identity standards api

### PR DESCRIPTION
**Pull request by NickGraham101**

### CODE REVIEW

File: collections/_development_standards/patterns.md [hunk 1]
```diff
@@ -169,3 +169,49 @@ Another positive side-effects of using CQRS is that there are naturally less mer
...
 * [CQRS - Microsoft](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs)
 * [CQRS - Martin Fowler](https://martinfowler.com/bliki/CQRS.html)
+
+
+### Managed Identities (MI)
+
+Using a managed identity allows an application to authenticate to any service that supports Azure AD authentication without having credentials.
+
+* Secretless configuration
+* Acquiring a bearer token for an API behund Azure AD authentication
```
Comment (DannyGB) [inline collections/_development_standards/patterns.md]: Typo:
```suggestion
* Acquiring a bearer token for an API behind Azure AD authentication
```

File: collections/_development_standards/patterns.md [hunk 2]
```diff
@@ -169,3 +169,49 @@ Another positive side-effects of using CQRS is that there are naturally less mer
...
+        # Add the token as a header
+        _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
+
+        # Make an API call
+        var response = await _httpClient.GetAsync("https://azurewebsites.net/resource/resourceid");
+    }
+
+#### References
+
+* [MI Overview - Microsoft](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
```
Comment (DannyGB) [inline collections/_development_standards/patterns.md]: I would add the root URL of the MS docs as a reference as it gives a bit more info and has further links to other useful resources for MI:
https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/

File: collections/_development_standards/patterns.md [hunk 3]
```diff
@@ -169,3 +169,49 @@ Another positive side-effects of using CQRS is that there are naturally less mer
 
 * [CQRS - Microsoft](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs)
 * [CQRS - Martin Fowler](https://martinfowler.com/bliki/CQRS.html)
+
+
+### Managed Identities (MI)
+
+Using a managed identity allows an application to authenticate to any service that supports Azure AD authentication without having credentials.
```
Comment (DannyGB) [inline collections/_development_standards/patterns.md]: I think there needs to be a more of an explanation as to why we want the devs to use MI. If you take a look at the other sections on this page they try to give enough background on the technology benefits to (hopefully) remove any knee-jerk negative reactions without having too much that we are just replicating the official docs.

**Review by DannyGB — CHANGES_REQUESTED** [review]
I'm wondering if this should be on a page of its own? Maybe a platform specific developer resource/ information page which we could then expand out with other Azure info as needed?

File: collections/_development_standards/patterns.md (hunk 1 above)
Comment (CloudPlatformer) [inline collections/_development_standards/patterns.md]:
```suggestion
* Acquiring a bearer token for an API behind Azure AD authentication
```

**Comment by NickGraham101** [comment]
> I'm wondering if this should be on a page of its own? Maybe a platform specific developer resource/ information page which we could then expand out with other Azure info as needed?

@DannyGB how about a separate page under Application Development Standards called Platform Integration?

File: collections/_development_standards/patterns.md (hunk 2 above)
Comment (NickGraham101) [reply collections/_development_standards/patterns.md]: Added link

File: collections/_development_standards/patterns.md (hunk 3 above)
Comment (NickGraham101) [reply collections/_development_standards/patterns.md]: Expanded section

**Review by DannyGB — APPROVED** [review]

**Pull request merged**